<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLogin</class>
 <widget class="QWidget" name="CLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLineEdit" name="EDIT_PWD">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>123456</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SET">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>295</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_LOGIN">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>210</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>登    录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>正在连接服务器，请稍等...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MIN">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CLOSE">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>admin</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>EDIT_PWD</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>BTN_LOGIN</tabstop>
  <tabstop>BTN_SET</tabstop>
  <tabstop>BTN_CLOSE</tabstop>
  <tabstop>BTN_MIN</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BTN_MIN</sender>
   <signal>clicked()</signal>
   <receiver>CLogin</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
